R/LearnerDensKDE.R

```diff
@@ -67,12 +67,19 @@ LearnerDensKDE = R6::R6Class("LearnerDensKDE",
         bw = bw,
         kernel = kernel))
 
+      dat <-  sapply(task$truth(), function (x, y) ((x - y) / bw), y = task$truth())
 
-      list(distr = Distribution$new(
-          name = paste(self$param_set$values$kernel, "KDE"),
-          short_name = paste0(self$param_set$values$kernel, "_KDE"),
-          pdf = pdf,
-          type = set6::Reals$new()), bw = bw, kernel = self$param_set$values$kernel)
+      pdfSquared2norm <- getFromNamespace(subset(distr6::listKernels(),
+                     ShortName == self$param_set$values$kernel)$ClassName, "distr6")$new()$pdfSquared2Norm(x = dat) / (length(task$truth())^2 * bw)
```

Comment:
Okay this makes sense to me if the value is always dependent on training data and not new values to make predictions on

Reply:
But `self$param_set$values$kernel` should be replaced with `kernel`

Reply:
Yes, the pdfSqaured2Norm only depends on the training data (observation data) 

but `kernel` output `Epan()`.  `self$param_set$values$kernel` have a default value so it will output the right kernel, right?

Reply:
You're right, ignore me!